frontend/src/main/scala/bloop/engine/tasks/ScalaNative.scala

```diff
@@ -0,0 +1,130 @@
+package bloop.engine.tasks
+
+import scala.util.{Failure, Success, Try}
+
+import java.net.URLClassLoader
+import java.nio.file.{Files, Path}
+import java.util.concurrent.ConcurrentHashMap
+
+import bloop.{DependencyResolution, Project}
+import bloop.cli.{ExitStatus, OptimizerConfig}
+import bloop.engine.State
+import bloop.exec.Forker
+import bloop.io.AbsolutePath
+import bloop.logging.Logger
+
+import monix.eval.Task
+
+object ScalaNative {
```

Comment:
Can we call this something else that clarifies more the meaning of the class? `ScalaNativeToolchain` for example?